sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/OpenCoverReportParserTest.java

```diff
@@ -129,16 +130,56 @@ public void branchCoverage() throws Exception {
         Assertions.entry(18, 1),
         Assertions.entry(25, 1));
 
-    assertThat(coverage.getBranchCoverage(filePath).size()).isEqualTo(5);
+    assertThat(coverage.getBranchCoverage(filePath))
+      .hasSize(5)
+      .contains(
+        new BranchCoverage(9, 4 , 2),
+        new BranchCoverage(12, 2 , 1),
+        new BranchCoverage(13, 2, 1),
+        new BranchCoverage(14, 4, 2),
+        new BranchCoverage(18, 6, 3));
```

Comment:
for the record, line 18 is
```
            return x < 2
                ? y < 3
                    ? z < 1
                        ? 1
                        : 2
                    : 3
                : 4;
```

3 out of 6 executed as all have value `0` - CORRECT